release-notes/9.0/supported-os.json

```diff
@@ -181,9 +190,8 @@
                         "Arm64",
                         "x64"
                     ],
-                    "supported-cycles": [
-                        "40",
-                        "39"
+                    "supported-versions": [
+                        "15.5"
```

Comment:
.NET 8+ cannot run on 12.5, due to requiring glibc 2.23.

From 12.5

$ ldd --version  
ldd (GNU libc) 2.22

Reply:
That's interesting, since according to https://github.com/dotnet/core/blob/v8.0.0/release-notes/8.0/supported-os.md it was supported in the past (while the document was already mentioning glibc 2.23).

Reply:
It was a mistake in .NET 8, then. I probably didn't notice it initially since I'm less familiar with Suse ecosystem. This time, I actually checked since we're doing a much more thorough job.